### PR TITLE
Show error instead of exception while creating the `--image-dir` folder

### PR DESCRIPTION
**Pull request by mschnitzer**

### CODE REVIEW

File: spec/unit/build_task_spec.rb [hunk 1]
```diff
@@ -144,6 +144,15 @@
...
       }.to raise_error(Machinery::Errors::BuildFailed, /i586/)
     end
+
+    it "shows an error when the current user does not have access to the image directory path" do
+      allow(LocalSystem).to receive(:validate_architecture)
+      allow_any_instance_of(Os).to receive(:architecture).and_return("x86_64")
+
+      expect {
+        build_task.build(system_description, "/test")
+      }.to raise_error(Machinery::Errors::BuildDirectoryCreateError)
```
Comment (aduffeck) [inline spec/unit/build_task_spec.rb]: It might make sense to verify that the path and the username are put out correctly.

**Comment by aduffeck** [comment]
lgtm otherwise

File: spec/unit/build_task_spec.rb (hunk 1 above)
Comment (mschnitzer) [inline spec/unit/build_task_spec.rb]: Agreed.

**Comment by mschnitzer** [comment]
@aduffeck implemented. Will squash that afterwards.

**Comment by aduffeck** [comment]
lgtm

**Comment by mschnitzer** [comment]
Superseded by #1677

**Pull request closed without merging**